### PR TITLE
My Requests Page

### PR DESCRIPTION
**Pull request by prescottprue**
### Description

* Adds My request page which links to request detail page
* Adds UI tests for request page

### Relevant Tickets (Please add `closes`, `refs`, etc)

- noted in https://docs.google.com/document/d/1P7DnvVSM_vsWN3-NbfMLBltSWGjjR3fJVY13MmU7ZTU

### Screenshots (if appropriate)

**Link in Navbar**
image

**Empty Messages**
image

**Requests Listed**
image

### CODE REVIEW

**Comment by thebitguru** [comment]
This has a missing index requests[createdBy ASC, status ASC].  Did you forget to include the updated indexes file?

File: src/containers/Navbar/AccountMenu.js [hunk 1]
```diff
@@ -52,6 +56,7 @@ function AccountMenu() {
         transformOrigin={{ vertical: 'top', horizontal: 'right' }}
         open={Boolean(anchorEl)}
         onClose={closeAccountMenu}>
+        <MenuItem onClick={goToMyRequests}>My Requests</MenuItem>
```
Comment (thebitguru) [inline src/containers/Navbar/AccountMenu.js:59]: FYI, we can use `component={Link} to={MY_REQUESTS_PATH} onClick={closeAccountMenu}`to have the menu entry act like an href.  This way, if desired, users can Cmd+Click to open in new tab, etc.  Not a big deal, but would be good to use going forward.

**Comment by prescottprue** [comment]
Oh, I hadn’t seen a warning for that. I can add it

**Review by thebitguru — APPROVED** [review]
FYI, the `requests` collection doesn't have the status and owner on it yet.  I will merge this PR and the request-details PR and then switch to the other collection.

File: src/containers/Navbar/AccountMenu.js (hunk 1 above)
Comment (prescottprue) [reply src/containers/Navbar/AccountMenu.js:59]: Agreed - I like doing this whenever possible,

**Pull request merged**